paf-mvp-cmp/src/controller.ts

```diff
@@ -449,9 +443,9 @@ export class Controller {
     document.cookie = `${
       this.config.siteOnlyCookieTcfCore
     }=${tcfCore.toString()};expires=${getPrebidDataCacheExpiration()}`;
-    removeCookie(Cookies.identifiers);
-    removeCookie(Cookies.preferences);
-    removeCookie(Cookies.lastRefresh);
+    OneKeyLib.removeCookie(Cookies.identifiers);
+    OneKeyLib.removeCookie(Cookies.preferences);
+    OneKeyLib.removeCookie(Cookies.lastRefresh);
```

Comment:
FAR One function for doing those 3 cleans would be cool

Reply:
Yes of course. There is a lot that needs to be adjusted in this controller, to better integrate with paf-lib